testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/DevelocityPluginSmokeTest.groovy

```diff
@@ -35,16 +35,19 @@ import java.util.concurrent.ConcurrentHashMap
 class DevelocityPluginSmokeTest extends AbstractSmokeTest {
 
     enum CI {
+        // https://github.com/etiennestuder/teamcity-build-scan-plugin/releases
         TEAM_CITY(
-            AbstractSmokeTest.TestedVersions.teamCityGradlePluginRef,
+            "v0.36",
             "https://raw.githubusercontent.com/etiennestuder/teamcity-build-scan-plugin/%s/agent/src/main/resources/init-scripts/develocity-injection.init.gradle"
         ),
+        // https://github.com/jenkinsci/gradle-plugin/releases
         JENKINS(
-            AbstractSmokeTest.TestedVersions.jenkinsGradlePluginRef,
+            "gradle-2.15",
             "https://raw.githubusercontent.com/jenkinsci/gradle-plugin/%s/src/main/resources/hudson/plugins/gradle/injection/init-script.gradle"
         ),
+        // https://github.com/gradle/develocity-bamboo-plugin/releases
         BAMBOO(
-            AbstractSmokeTest.TestedVersions.bambooGradlePluginRef,
+            "develocity-bamboo-plugin-3.0.1",
```

Comment:
💅 latest there is [3.0.2 aka develocity-bamboo-plugin-3.0.2](https://github.com/gradle/develocity-bamboo-plugin/releases/tag/develocity-bamboo-plugin-3.0.2)

Could we automate this as well?

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases